nxt/locator.py

```diff
@@ -126,7 +126,7 @@ def find(
     :type host: str or None
     :param filters: Additional filter keywords or backends parameters.
     :return: The found brick, or an iterator if `find_all` is ``True``
-    :rtype: Brick or Iterator[Brick]
+    :rtype: nxt.brick.Brick or Iterator[Brick]
```

Comment:
Can you also change the Brick in the Iterator[]?

Thanks

Reply:
Thank you for the feedback. I improve the error and oh yes I change that too. On that occasion I also scroll through again and see if something like that can be changed in one more place.